### ISSUE TITLE
Adult_Females is not an exported object

### ISSUE THREAD

**Issue by SimonDedman** [SimonDedman/gbm.auto#54]
> library(devtools)
Loading required package: usethis
> document()
Updating gbm.auto documentation
Updating roxygen version in /home/simon/Dropbox/Galway/Analysis/R/gbm.auto/DESCRIPTION
Loading gbm.auto
Error: 'Adult_Females' is not an exported object from 'namespace:gbm.auto'

Can't update man pages because of this.
https://stackoverflow.com/questions/40364061/data-not-exported-from-namespace-in-r/49157929#49157929

**Comment by SimonDedman** [comment]
error could not find function
https://stackoverflow.com/questions/7027288/error-could-not-find-function-in-r
potentially related to this issue.
worksround is prepend function calls with `gbm.auto:::`

**Comment by SimonDedman** [comment]
Matt Jargowski comment, "I did try and troubleshoot the issue a bit (by downloading older versions of R) and I believe the issue is with the package version 1.2.1. The package worked fine on Amanda Jefferson's computer, she had version 1.2.0; however, I uninstalled it on her computer and then tried to re-install the package and she now has the same issue as myself".
v1.2.0 is commit ".", last one on 9/2/19: https://github.com/SimonDedman/gbm.auto/commit/2c303d8ee8f1da278bcd33e593f96149709f7f94
v1.2.1 is 26/11/19: https://github.com/SimonDedman/gbm.auto/commit/941bcf6c25e4479b54cb4545486a3c613d5021bd
So it's somewhere between those?
Highly likely to be "Backlog push." commit, https://github.com/SimonDedman/gbm.auto/commit/b9321f6b4721b8caa4d87b1044f3e72ea94ff3ed maybe in combo with subsequent "delete old data folder files" commit https://github.com/SimonDedman/gbm.auto/commit/1bded10cc4ed2e96f56440f2f199cdc27ccddd20: first one saw 6 RData files added & 8 R files (gbm.auto, e.g.) deleted from being exported in NAMESPACE. Doubt I would have done that myself since NAMESPACE isn't edited by hand, was there a packaging change between Feb 25 & Nov 26 2019?
"Backlog push" saw Data folder added and RData files entered there.  "delete old data folder files" saw data folder and its contents deleted. Maybe duplicate Data as data and see what happens? Can't imagine this'll change the R scripts to be exported again though?
